tcl/target/gd32vw55x.cfg

```diff
@@ -0,0 +1,36 @@
+#
+# GigaDevice GD32VW55x target
+#
+# https://www.gigadevice.com/products/microcontrollers/gd32/risc-v/
+#
+set _CHIPNAME riscv
+jtag newtap $_CHIPNAME cpu -irlen 5 -expected-id 0x10307a6d
+
+set _TARGETNAME $_CHIPNAME.cpu
+target create $_TARGETNAME riscv -chain-position $_TARGETNAME
+$_TARGETNAME configure -work-area-phys 0x20000000 -work-area-size 20480 -work-area-backup 0
+
+
+# Work-area is a space in RAM used for flash programming
+if { [info exists WORKAREASIZE] } {
```

Comment:
Don't see any use for `_WORKAREASIZE `, line 11 has work-area-size set to 20480.